### PR TITLE
Update WG Structured logging 2021 annual report

### PR DESCRIPTION
**Pull request by serathius**
cc @pohly @dims 
Fixes #https://github.com/kubernetes/community/issues/6393

### CODE REVIEW

File: wg-structured-logging/annual-report-2021.md [hunk 1]
```diff
@@ -5,50 +5,50 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - Graduated [Structured Logging](https://github.com/kubernetes/enhancements/issues/1602) to Beta
+   - Graduated [Derecation of klog specific flags](https://github.com/kubernetes/enhancements/issues/2845) to Beta
+   - Created draft proposal for [Contextual logging](https://docs.google.com/document/d/1FaDj3_MvLWSuUtlVapgfTUglhYjBem1LNcpdd_OyBwU/edit?usp=sharing)
```
Comment (pohly) [inline wg-structured-logging/annual-report-2021.md:10]: The KEP PR was also created in 2021, I think that's a better link than the Google doc. The prototype also was ready ->

```suggestion
   - Created prototype and KEP for [Contextual logging](https://github.com/kubernetes/enhancements/pull/3078)
```

File: wg-structured-logging/annual-report-2021.md [hunk 2]
```diff
@@ -5,50 +5,50 @@
...
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - Continiusly migrating kubernetes/kubernetes repository to structured logging. In 2021 kubelet, kube-scheduler, kube-proxy were migrated.
```
Comment (pohly) [inline wg-structured-logging/annual-report-2021.md:14]:
```suggestion
   - Continuously migrating kubernetes/kubernetes repository to structured logging. In 2021 kubelet, kube-scheduler, kube-proxy were migrated.
```

File: wg-structured-logging/annual-report-2021.md [hunk 3]
```diff
@@ -5,50 +5,50 @@
...
 
 ## Project health
 
 1. What's the current roadmap until completion of the working group?
 
-   -
-   -
-   -
+   - Graduate [Contextual Logging](https://github.com/kubernetes/enhancements/issues/3077) to Beta and GA
+   - Graduate [Derecation of klog specific flags](https://github.com/kubernetes/enhancements/issues/2845) to GA
```
Comment (pohly) [inline wg-structured-logging/annual-report-2021.md:22]:
```suggestion
   - Graduate [Deprecation of klog specific flags](https://github.com/kubernetes/enhancements/issues/2845) to GA
```

File: wg-structured-logging/annual-report-2021.md [hunk 4]
```diff
@@ -5,50 +5,50 @@
...
+   - Graduated [Structured Logging](https://github.com/kubernetes/enhancements/issues/1602) to GA
+   - All code in kubernetes/kubernetes repository is migrated to Structured Logging API
 
 2. Does the group have contributors from multiple companies/affiliations?
 
-   -
+   - Yes, current members are from 1 ZTE, 1 Appdynamics, 1 Google, 3 VMware, 1 Intel and 2 unaffiliated.
```
Comment (pohly) [inline wg-structured-logging/annual-report-2021.md:28]:
```suggestion
   - Yes, current members are from 1 ZTE, 1 AppDynamics, 1 Google, 3 VMware, 1 Intel and 2 unaffiliated.
```

**Review by pohly — CHANGES_REQUESTED** [review]

File: wg-structured-logging/annual-report-2021.md (hunk 1 above)
Comment (shivanshu1333) [reply wg-structured-logging/annual-report-2021.md:10]: agree

File: wg-structured-logging/annual-report-2021.md [hunk 5]
```diff
@@ -5,50 +5,50 @@
 1. What work did the WG do this year that should be highlighted?
    For example, artifacts, reports, white papers produced this year.
 
-   -
-   -
-   -
+   - Graduated [Structured Logging](https://github.com/kubernetes/enhancements/issues/1602) to Beta
+   - Graduated [Derecation of klog specific flags](https://github.com/kubernetes/enhancements/issues/2845) to Beta
```
Comment (shivanshu1333) [inline wg-structured-logging/annual-report-2021.md:9]:
```suggestion
   - Graduated [Deprecation of klog specific flags](https://github.com/kubernetes/enhancements/issues/2845) to Beta
```

**Review by pohly — APPROVED** [review]
/lgtm

**Comment by jdumars** [comment]
/approve
/hold
If no objections within 72 hours from the time of this comment, submitting WG can unhold at their discretion.

**Comment by dims** [comment]
@serathius give me a couple of days, will review and then we can merge. thanks!

**Pull request merged**